src/settings/LanguagePicker.js

```diff
@@ -20,24 +20,12 @@ export default class LanguagePicker extends PureComponent<Props> {
   static contextType: Context<GetText> = TranslationContext;
   context: GetText;
 
-  getTranslatedLanguages: () => Language[] = () =>
-    languages.map((language: Language) => {
-      const _ = this.context;
-      const translatedName = _(language.name);
-      return {
-        ...language,
-        name: translatedName,
-      };
-    });
-
-  getFilteredLanguageList: string => Language[] = filter => {
-    const list = this.getTranslatedLanguages();
-
+  getFilteredLanguageList: string => $ReadOnlyArray<Language> = filter => {
     if (!filter) {
-      return list;
+      return languages;
     }
 
-    return list.filter(item => {
+    return languages.filter(item => {
```

Comment:
This change doesn't seem right -- it means that the search will look at the English name and selfname, instead of the current UI language's name and selfname.

The code around here could be improved in general, but probably the best thing for the current task is to keep the change to this code simple and local: just handle `subtitle=` below in the same way as `title=`, without changing these other functions.

Reply:
Indeed! Thanks for catching this.